awx/ui/src/screens/Job/JobOutput/JobOutput.js

```diff
@@ -139,6 +139,7 @@ function JobOutput({ job, eventRelatedSearchableKeys, eventSearchableKeys }) {
       setJobTreeReady: () => setIsTreeReady(true),
     },
     job.id,
+    job.related?.children_summary,
```

Comment:
the child summary parameter shouldn't be necessary, since it will always be in flat mode for job types that don't have child summaries